src/rgw/setup/rgw.py

```diff
@@ -1090,9 +1103,12 @@ def _update_resource_limit_based_config(conf: MappedConf, client_section: str):
         tuned_memory_val = const.SVC_MEM_FACTOR * (input_max_mem_limit_val - initial_startup_mem_val) / mem_per_thread_pre_req_val
         tuned_cpu_val = input_max_cpu_limit_val / cpu_per_thread_val
 
-        thread_pool_size_val = math.floor(min(tuned_memory_val, tuned_cpu_val))
-        concurrent_max_requests_val = math.floor(min(tuned_memory_val, 2*tuned_cpu_val))
-        if thread_pool_size_val > 0 :
+        if concurrent_max_requests_val is None:
```

Comment:
if comment of line 1068 is addressed, this extra checks wont be needed!

Reply:
this is required where we are handling if any of the values that are not specified in solution.yaml then we are assigning generated value.

Reply:
yes i understand, as i stated we can handle at one place, check comment at line 1068